modules/renderer/features.js

```diff
@@ -95,104 +95,104 @@ export function rendererFeatures(context) {
             defaultMax: (max || Infinity),
             enable: function() { this.enabled = true; this.currentMax = this.defaultMax; },
             disable: function() { this.enabled = false; this.currentMax = 0; },
-            hidden: function() { return !context.editable() || this.count > this.currentMax * _cullFactor; },
-            autoHidden: function() { return this.hidden() && this.currentMax > 0; }
+            hidden: function() { return !this.enabled || !context.editable() || this.count > this.currentMax * _cullFactor; },
+            autoHidden: function() { return this.hidden() && this.enabled; }
```

Comment:
I think the changes on 98-99 would change how the filtering menu works.. `enabled` is whether the user has deliberately checked the check box, but `hidden` and `autoHidden` will update the checkboxes if the user zooms out or a feature like buildings or points is getting culled because there are too many of them.

Reply:
Oh okay. There was an issue where `hidden()` would return `true` if `count` was `0` even if `enabled` was `false`. In the latest commit I restored the old behavior while accounting for this case.